demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -718,6 +712,17 @@ def is_valid_file_type(self, file_type: FileType, file_path: str):
 
         return True
 
+    def is_skipped_file(self, file_path: str) -> bool:
+        """check wether the file in the given file_path is in the SKIPPED_FILES list.
+
+        Args:
+            file_path: the file on which to run.
+
+        Returns:
+            bool. true if file is in SKIPPED_FILES list, false otherwise.
+        """
+        return Path(file_path).name in SKIPPED_FILES
```

Comment:
what happens when one modifies `CommonServerPython.py`? Is it not validated? Perhaps worth checking the parent as well in case it's in that list.

Reply:
from the modified code it looks like it wasn't validated until now

Reply:
Can we fix this broken window easily?